log/print.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 )
 
-func printf(severity Severity, withTime bool, format string, v ...interface{}) {
+func printf(severity Severity, withTime bool, format string, v ...interface{}) string {
```

Comment:
Function name is confusing at first read as Printf is already known.

Reply:
Indeed, however, this function is preexisting and I can't come up with a better name at this point.